### PR TITLE
Bug 1753265: Add 'Stopping' status for Virtual Machines

### PR DESCRIPTION
**Pull request by irosenzw**
Signed-off-by: Ido Rosenzwig <irosenzw@redhat.com>

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts [hunk 1]
```diff
@@ -51,6 +57,30 @@ const isBeingMigrated = (vm: VMKind, migrations?: K8sResourceKind[]): VMStatus =
   return NOT_HANDLED;
 };
 
+const isBeingStopped = (vm: VMKind, launcherPod: PodKind = null): VMStatus => {
+  if (isVMReady(vm) || isVMCreated(vm)) {
+    const podStatus = getPodStatus(launcherPod);
+    const ContainerStatuses = getPodContainerStatuses(launcherPod);
```
Comment (vojtechszocs) [inline frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts]: Nit: why leading uppercase on variable name?

File: frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts [hunk 2]
```diff
@@ -51,6 +57,30 @@ const isBeingMigrated = (vm: VMKind, migrations?: K8sResourceKind[]): VMStatus =
...
+    if (ContainerStatuses) {
```
Comment (vojtechszocs) [inline frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts]: Nit: we can reduce cyclomatic complexity

```ts
if (!ContainerStatuses) return NOT_HANDLED;

// ..
```

File: frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts (hunk 1 above)
Comment (irosenzw) [reply frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts]: Done.

File: frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts (hunk 2 above)
Comment (irosenzw) [reply frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts]: Done.

**Pull request merged**